python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+from bigdl.chronos.forecaster.base_forecaster import BasePytorchForecaster
+from bigdl.chronos.model.autoformer import model_creator
+from torch.utils.data import TensorDataset, DataLoader
+from bigdl.chronos.model.autoformer.Autoformer import AutoFormer
+import torch.nn as nn
+import numpy as np
+import os
+
+
+class AutoformerForecaster(BasePytorchForecaster):
+    def __init__(self,
+                 seq_len,
+                 label_len,
+                 pred_len,
+                 enc_in,
+                 freq,
+                 dec_in,
+                 c_out,
+                 output_attention=False,
+                 moving_avg=25,
+                 d_model=512,
+                 embed='timeF',
+                 dropout=0.05,
+                 factor=3,
+                 n_head=8,
+                 d_ff=2048,
+                 activation='gelu',
+                 e_layer=2,
+                 d_layers=1):
```

Comment:
I think we need parameter document for these parameters like https://github.com/intel-analytics/BigDL/blob/main/python/chronos/src/bigdl/chronos/forecaster/tcn_forecaster.py#L55-L95

You may find some explanation in https://github.com/zhouhaoyi/Informer2020#usage

Some parameters can be renamed to align with other forecasters (e.g. `seq_len` = `past_seq_len`, `pred_len` = `future_seq_len`...)